src/c++/vdo/base/vio.c

```diff
@@ -307,6 +307,7 @@ void vio_record_metadata_io_error(struct vio *vio)
  * make_vio_pool() - Create a new vio pool.
  * @vdo: The vdo.
  * @pool_size: The number of vios in the pool.
+ * @block_count: The number of blocks (VDO_BLOCK_SIZE) per vio.
  * @thread_id: The ID of the thread using this pool.
```

Comment:
I'd cut "(VDO_BLOCK_SIZE)", it's not super helpful without context. Maybe say "4k blocks" if you're worried about confusion? I think "pages" is the right term , but we don't use that elsewhere so it's probably also not that helpful in context.

Reply:
switching to "4k blocks" ... "pages" to me pretty strongly suggests hardware memory pages unless we say otherwise